src/server.ts

```diff
@@ -1,47 +1,98 @@
 import * as fsPath from 'path';
+import * as http from 'http';
 import * as nunjucks from 'nunjucks';
 
 import {Pod} from './pod';
 import {StaticRoute} from './router';
+import {Watcher} from './watcher';
 
 import express = require('express');
 
 export class Server {
-  constructor() {}
-}
+  private pod: Pod;
+  private app?: express.Application;
+  private httpServer?: http.Server;
+  private watcher: Watcher;
+  private port: string | number;
+
+  constructor(pod: Pod, port: string | number) {
+    this.pod = pod;
+    this.port = port;
+    this.watcher = new Watcher(pod, this);
+    this.app = undefined;
```

Comment:
initializing to `undefined` is unnecessary. can you call createApp() here? assuming this isn't a public (outside of amagaki) class, you should be able to determine if an app is _always_ needed or not.

Reply:
Removed the superfluous inits.

Scott might use the `Server` object within https://github.com/blinkk/amagaki-starter/pull/10 -- is it OK to leave the `start` as a different step outside of the constructor?

Reply:
it's fine to leave "start" outside, but if an "app" is almost always guaranteed to be needed by a server object, let's just create it here in the constructor.

Reply:
nevermind, i see that an app needs to be re-initialized when the restartPod() method is called, and per another comment this is moot anyway since it's been moved to a local var.